### DISCUSSION TITLE
[Feature Request] Hotkey Detection Mod Hook

### DISCUSSION THREAD

**Discussion by WW-TC** [SmartlyDressedGames/Unturned-3.x-Community#4543] [Ideas]
A simple mod hook that lets you do extra stuff using hotkeys would make it good for additional features for vehicles or items.

**Comment by Jdance-Media** [comment]
Of course, replication on the server is a tad bit harder to implement, but at the very least having an input for a KeyCode to watch on Update is good enough for many things.
